enhancements/installer/openstack-no-fip-install.md

```diff
@@ -0,0 +1,136 @@
+---
+title: openstack-bring-your-own-external-connectivity
+authors:
+  - "@egarcia"
+  - "@adduarte"
+reviewers:
+  - "@mandre"
+  - "@pierre"
+approvers:
+  - "@mandre"
+  - "@pierre"
+creation-date: 2020-5-26
+last-updated: 2020-5-26
+status: implementable
+---
+
+# OpenStack BYO External Connectivity
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [x] Design details are appropriately documented from clear requirements
+- [x] Test plan is defined
+- [x] Graduation criteria
+- [ ] User-facing documentation is created in [openshift/docs]
+
+## Summary
+
+Numerous customers have requested the ability to run the OpenShift on OpenStack IPI installer without using Floating IP addresses.
+In the installer, we use floating IPs as a way to set up exeternal connectivity to the OpenShift cluster. Depending on how the OpenStack cluster
+is set up, the use of floating IPs may be impossible or very inconvenient. In order to support these customers, we intend to support installations
```

Comment:
>  In order to support these customers, we intend to support installations on these types of OpenStack clusters.

to me this line is really confusing esp `these` are exactly are `these types`? Is it where `floating IPs may be impossible or very inconvenient` or `where there are is X alternative available for load balancing` ?

Reply:
I think you can just drop the "In order to support these customers, "